mlflow/store/tracking/dbmodels/models.py

```diff
@@ -72,11 +72,11 @@ class SqlExperiment(Base):
     Lifecycle Stage of experiment: `String` (limit 32 characters).
                                     Can be either ``active`` (default) or ``deleted``.
     """
-    creation_time = Column(BigInteger(), default=int(time.time() * 1000))
+    creation_time = Column(BigInteger, default=int(time.time() * 1000))
```

Comment:
Is `BigInteger()` equivalent to `BigInteger`?

Reply:
yep!